test/Maestro.ScenarioTests/MaestroScenarioTestBase.cs

```diff
@@ -260,19 +260,23 @@ await WaitForUpdatedPullRequestAsync(targetRepoName, targetBranch)
         public async Task CheckBatchedAzDoPullRequest(string source1RepoName, string source2RepoName, string targetRepoName, string targetBranch,
             List<Microsoft.DotNet.DarcLib.DependencyDetail> expectedDependencies, string repoDirectory, bool complete = false)
         {
-            string expectedPRTitle = $"[{targetBranch}] Update dependencies from {_parameters.AzureDevOpsAccount}/{_parameters.AzureDevOpsProject}/{source1RepoName} {_parameters.AzureDevOpsAccount}/{_parameters.AzureDevOpsProject}/{source2RepoName}";
-            await CheckAzDoPullRequest(expectedPRTitle, targetRepoName, targetBranch, expectedDependencies, repoDirectory, complete, true, null, null);
+            string[] expectedPRTitles =
```

Comment:
Suggestion: we have just two combinations, but listing all expected PR combinations doesn't capture the intent and wouldn't scale for multiple repos.

What about something like this?
```csharp
pullRequest.Title.starstsWith("[{targetBranch}] Update dependencies from")
foreach (string repoName in [source1RepoName, source2RepoName]) {
    pullRequest.Title.contains($"{_parameters.AzureDevOpsAccount}/{_parameters.AzureDevOpsProject}/{repoName}");
}
```